docs/content/docs/operators/feature/binarizer.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Binarizer"
+weight: 1
+type: docs
+aliases:
+- /operators/feature/binarizer.html
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Binarizer
+
+Binarizer binarizes the columns of continuous features by the given thresholds.
+The continuous features may be DenseVector, SparseVector, or Numerical Value.
+
+### Input Columns
+
+| Param name | Type          | Default | Description                     |
+| :--------- | :------------ | :------ |:--------------------------------|
+| inputCols  | Number/Vector | `null`  | Number/Vectors to be binarized. |
+
+### Output Columns
+
+| Param name | Type          | Default | Description               |
+|:-----------|:--------------|:--------|:--------------------------|
+| outputCols | Number/Vector | `null`  | Binarized Number/Vectors. |
+
+### Parameters
+
+| Key         | Default   | Type     | Required | Description                                          |
+|-------------|-----------|----------|----------|------------------------------------------------------|
```

Comment:
The table format seems different from the above. Does it behaves similarly as above?